server.ts

```diff
@@ -221,7 +244,7 @@ app.get("/funds/:id", (req, res) => {
   } catch (error) {
     res.sendStatus(500);
   }
-});
+})
```

Comment:
maybe we should start using a uniform formatting tool like Prettier so these small changes are not needed